### PR TITLE
Dev/franka test targets

### PR DESCRIPTION
**Pull request by frkndmn**
target files of franka

### CODE REVIEW

**Comment by sasilva1998** [comment]
@frkndmn I also notices these two files `listOfCommands.txt` and `mycommands.txt`. What is their purpose? otherwise I would say they can be deleted. Then the rest is fine.

**Review by sasilva1998 — CHANGES_REQUESTED** [review]

File: ros-franka_common/install.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+# Check if robotnik libraries have not been installed yet
+if [ ! -d "/opt/ros/${IROHMS_ROS_DISTRO}/lib/robotnik_twist2ackermann" ]
```
[the remark this answers is not in the document]
Comment (frkndmn) [reply ros-franka_common/install.yaml]: Hi Steven, you are right about both comments. listofcommands.txt and mycommands.txt are not required. Also, I didn't notice IROHMS. It should be changed. Should I fix these problems and after that should I pull the request again?

[the remark this answers is not in the document]
Comment (sasilva1998) [reply ros-franka_common/install.yaml]: Hi Furkan, you dont have to do a new pull request, just upload a commit with those changes to the branch and they will be combined in the current one

[the remark this answers is not in the document]
Comment (sasilva1998) [reply ros-franka_common/install.yaml]: @frkndmn saw that the package `ros-franka_common` is not there anymore, did you delete it?

[the remark this answers is not in the document]
Comment (sasilva1998) [reply ros-franka_common/install.yaml]: I ask because I dont see a commit for that, if it is not needed it is okay then

**Pull request merged**